src/other/utils.jl

```diff
@@ -246,7 +248,7 @@ function _findall(B::BitVector)::Union{UnitRange{Int}, Vector{Int}}
         end
         if c == ~UInt64(0)
             if stop != -1
-                I = Vector{Int}(undef, nnzB)
+                I = Vector{Int}(undef,nnzB)
                 I[1:i-1] .= start:stop
```

Comment:
Broadcasting is still used here. Is it really a problem to use it in general, though? This function should be compiled only once for `Vector{Bool}` and precompilation should work well, so latency probably doesn't matter a lot.

Reply:
It's already fixed in main, #2771.
The problem is not with latency but with allocations.

Reply:
@nalimilan - in particular https://github.com/JuliaData/DataFrames.jl/pull/2771#issuecomment-847931350 shows that something bad is happening with Julia compiler in this case. Since adding a loop is not that much longer and is a "safe choice" I accepted #2771.

Anyway - are you OK with merging this PR? (as the issue you asked about is unrelated) Thank you!